susemanager-utils/testing/automation/update-obs-environment-cr-project.sh

```diff
@@ -1,15 +1,93 @@
 #!/bin/bash
```

Comment:
Unrelated to the PR, but maybe we should consider exiting if there are errors.

Reply:
I considered this, but then when I check if a project exists, I would need something like ```osc ls PROJECT || true``` and then I would need to set pipefails ... I think is clearer the way it is.

Reply:
You can disable exiting on errors when needed with `set` without any other aditional magic, and keep the "exit on error" for the general case.